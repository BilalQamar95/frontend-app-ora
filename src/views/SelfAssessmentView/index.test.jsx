import { shallow } from '@edx/react-unit-test-utils';
import { AssessmentView } from '.';

jest.mock('./AssessmentContentLayout', () => 'AssessmentContentLayout');

jest.mock('./hooks', () => jest.fn().mockReturnValue({
  submission: 'submission',
  oraConfigData: 'oraConfigData',
}));

describe('<AssessmentView />', () => {
  it('renders', () => {
    const wrapper = shallow(<AssessmentView />);
    expect(wrapper.snapshot).toMatchSnapshot();
  });
});
