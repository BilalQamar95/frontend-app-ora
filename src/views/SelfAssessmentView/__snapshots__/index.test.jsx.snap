// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AssessmentView /> renders 1`] = `
<Fragment>
  <ProgressBar />
  <AssessmentContentLayout
    oraConfigData="oraConfigData"
    submission="submission"
  />
</Fragment>
`;
